index.js

```diff
@@ -28,46 +28,40 @@ module.exports = function (babel, args) {
       '@babel/plugin-proposal-class-properties',
       '@babel/plugin-proposal-optional-chaining',
     ],
+    env: {
+      browser: {
+        presets: [
+          ['@babel/preset-env', {
+            targets: options.browsers || '> 2% in US',
+            modules: false,
+            useBuiltIns: 'usage',
+          }],
+        ],
+        plugins: [
+          ['module:fast-async', { spec: true }],
+          '@babel/plugin-transform-react-constant-elements',
+          '@babel/plugin-transform-react-inline-elements',
+          'babel-plugin-transform-react-remove-prop-types',
+          'babel-plugin-transform-react-class-to-function',
+        ],
+      },
+      webserver: {
```

Comment:
Does this mean BABEL_ENV needs to be "webserver"?

Reply:
If you are planning on rendering React with CSS modules server-side, yes. I've added detail to the PR's description.

Reply:
As a matter of fact, I could even break this part of the config out into another branch if we wanted. The default exclusion of css module transforms is a bigger priority than the rest. It will allow us to build react-components as ES modules using our babel preset. I currently have it working with a whole new babel config and would like to use this one.

Reply:
I would think that should be the default then, rather than "webserver". But if you're saying we're no longer going to do this, then maybe just bump major and remove it.

Reply:
Also, some of these changes point to the question of whether we should just have babel-preset-gasbuddy-react that uses babel-preset-gasbuddy + stuff.

Reply:
I like the idea of having a react preset. I might veer in that direction. As far as the `webserver` environment, I was suggesting we use the env property because it's a supported method for enabling plugins, but I don't believe there's an easy way to disable plugins. Hence why we had to create a whole new Babel config.